packages/react-dom-bindings/src/server/ReactDOMServerFormatConfig.js

```diff
@@ -156,6 +158,7 @@ export function createResponseState(
   bootstrapScriptContent: string | void,
   bootstrapScripts: $ReadOnlyArray<string | BootstrapScriptDescriptor> | void,
   bootstrapModules: $ReadOnlyArray<string | BootstrapScriptDescriptor> | void,
+  externalRuntimeConfig: string | BootstrapScriptDescriptor | void,
```

Comment:
I think it makes sense to support the same configuration options as bootstrapScripts?